src/App.js

```diff
@@ -37,30 +36,29 @@ const App = () => {
   chrome?.updateDocumentTitle?.('Hybrid Cloud Console Home', false);
 
   useEffect(() => {
+    console.log(__webpack_share_scopes__.default);
```

Comment:
Sorry, this is a left over from the debugging I was doing. It does not belong here.